src/main/java/vazkii/psi/common/block/base/ModBlocks.java

```diff
@@ -64,21 +64,18 @@ public class ModBlocks {
 	@SubscribeEvent
 	public static void register(RegisterEvent evt) {
 		evt.register(ForgeRegistries.Keys.BLOCKS, helper -> {
-			BlockBehaviour.Properties metalBlockProperties = BlockBehaviour.Properties.of().mapColor(MapColor.METAL).instrument(NoteBlockInstrument.IRON_XYLOPHONE).requiresCorrectToolForDrops().strength(5.0F, 6.0F).sound(SoundType.METAL);
-			BlockBehaviour metalBehaviour = new Block(metalBlockProperties);
-
-			cadAssembler = new BlockCADAssembler(Block.Properties.copy(metalBehaviour).strength(5, 10).sound(SoundType.METAL).noOcclusion());
-			programmer = new BlockProgrammer(Block.Properties.copy(metalBehaviour).strength(5, 10).sound(SoundType.METAL).noOcclusion());
+			cadAssembler = new BlockCADAssembler(Block.Properties.of().mapColor(MapColor.METAL).instrument(NoteBlockInstrument.IRON_XYLOPHONE).requiresCorrectToolForDrops().strength(5, 10).sound(SoundType.METAL).noOcclusion());
```

Comment:
Note: Blocks don't seem to have proper tools set up for breaking, conjured blocks are no longer instant punch

Reply:
@Kamefrede addet it in 85ef344 
I don't know what his goal was but I think there should be at least a Tool that breaks them faster. Fixing it is not a problem but I neat to know what the goal is.

Reply:
I removed the tool requirement.